### PR TITLE
docs: tx-helper

### PR DESCRIPTION
**Pull request by ryanleecode**
Docs for the newly added tx-helper subpath

see: https://github.com/polkadot-js/extension/pull/1389 for usage

### CODE REVIEW

File: packages/light-client-extension-helpers/README.md [hunk 1]
```diff
@@ -211,4 +211,45 @@ client.supervise()
 
 ### tx-helper
 
-TODO: write usage guide
+The tx-helper package allows you to easily sign transactions. You just need to calldata and an implementation of the
```
Comment (marshacb) [inline packages/light-client-extension-helpers/README.md:214]: Should this be `You just need calldata`?

**Review by marshacb — APPROVED** [review]
lgtm!

File: packages/light-client-extension-helpers/README.md (hunk 1 above)
Comment (ryanleecode) [reply packages/light-client-extension-helpers/README.md:214]: no u need a signer implementation too

Comment (ryanleecode) [reply packages/light-client-extension-helpers/README.md:214]: for example: https://github.com/polkadot-js/extension/pull/1389/files#diff-03379f3e1169c9b022a9b6ece3a998003b83b9f9c1417243026c8ef722c23902R78

Comment (marshacb) [reply packages/light-client-extension-helpers/README.md:214]: Oh yea, I understand that. Im asking should the sentence read `You just need calldata...` as opposed to `You just need to calldata...`

Comment (ryanleecode) [reply packages/light-client-extension-helpers/README.md:214]: oh oops :)

**Review by Pietrek96 — APPROVED** [review]

**Pull request merged**